conda_gitenv/tests/integration/test_resolve.py

```diff
@@ -8,18 +8,48 @@
 
 
 class Test_full_build(unittest.TestCase):
-    def test_basic_env(self):
+    def check_basic_env(self):
         repo = setup_samples.basic_repo()
```

Comment:
Yeah, that's my thinking ... before it was only one test, but now there are two, both of which are using the **same** framework to create a test repo.

I think its the first obvious place for me to look.